examples/Patterns/Saga/TransferProcess.cs

```diff
@@ -146,16 +146,16 @@ private async Task AwaitingCreditConfirmation(IContext context)
             switch (context.Message)
             {
                 case Started _:
-                    // if we are in this state when started then we need to recreate the TryCredit actor
+                    // if we are in this state when started then we need to recreate the TryCredit actorew
                     context.SpawnNamed(TryCredit(_to, +_amount), "CreditAttempt");
                     break;
                 case OK msg:
-                    decimal fromBalance = await _from.RequestAsync<decimal>(new GetBalance(), TimeSpan.FromMilliseconds(2000));
-                    decimal toBalance = await _to.RequestAsync<decimal>(new GetBalance(), TimeSpan.FromMilliseconds(2000));
+                    decimal fromBalance = await  context.RequestAsync<decimal>(_from, new GetBalance(), TimeSpan.FromMilliseconds(2000));
```

Comment:
this is me guessing how to fix this. Never done protoactor before, please check!

Reply:
This is correct. Basically, before we used to have two ways of doing Tell and Request - either directly on the PID and via a context. With the context it's possible to have contextual data passed through the chain, allowing for e.g. tracing headers to be propagated between actors and applications. The PID however didn't support this, so we killed it off in favor of only supporting context.